### ISSUE TITLE
CRFrnn_layer batch size

### ISSUE THREAD

**Issue by maloletnik** [sadeepj/crfasrnn_keras#6]
Hi,

Your implementation of the crf layer supports only batch_size = 1.
Correct?

**Comment by sadeepj** [comment]
That is correct. This is now mentioned under "Limitations" in the README. I'll leave this issue open so that I'll remember to fix it. If you have a patch to fix this, you're more than welcome to submit a pull request.

**Comment by runzeer** [comment]
thx for your excellent work! Now I have a tensorflow framework.So can I use the crfrnnlayer as a layer behind the layers from tensorflow such as slim?

**Comment by mitra052** [comment]
Has the batch_size issue been fixed?

**Comment by pvthuy** [comment]
Did anyone fix this?
